docs/faqs.md

```diff
@@ -403,3 +403,59 @@ colima start --arch x86_64  --cpu 4 --memory 8 --disk 10
 Colima will show the path to the docker socket so I have a
 [Coder template](./docker-code-server/main.tf) that prompts the Coder admin to
 enter the docker socket as a Terraform variable.
+
+## How to make a `coder_app` optional?
+
+An example use case is the user should decide if they want a browser-based IDE
+like code-server when creating the workspace.
+
+1. Create a `coder_parameter` as a `bool` to ask the user if they want the
+   code-server IDE
+
+```hcl
+data "coder_parameter" "code_server" {
+  name        = "code-server (optional)"
+  description = "Use VS Code in a browser"
+  type        = "bool"
+  default     = false
+  mutable     = true
+  icon        = "/icon/code.svg"
+  order       = 6
+}
+```
+
+2. Add conditional logic to the `startup_script` to install and start
+   code-server if the `bool` is true
+
+```sh
+# install and code-server, VS Code in a browser
+
+if [ ${data.coder_parameter.code_server.value} = true ]; then
+  echo "🧑🏼‍💻 Downloading and installing the latest code-server IDE..."
+  curl -fsSL https://code-server.dev/install.sh | sh
+  code-server --auth none --port 13337 >/dev/null 2>&1 &
+fi
+```
+
+3. Add a Terraform meta-argument `count` in the `coder_app` resource so it will
+   only create the resource if the `coder_parameter` is true
```

Comment:
```suggestion
3. Add a Terraform meta-argument [`count`](https://developer.hashicorp.com/terraform/language/meta-arguments/count) in the `coder_app` resource so it will
   only create the resource if the `coder_parameter` is `true`
```